Lesson16/bilge/script.js

```diff
@@ -0,0 +1,416 @@
+ ... some code
+} else {
+ ... some code
+}
+
+Switch statement
+switch (day) {
+  case 'Monday':  // this means that day === 'Monday'
+   console.log('It\'s Monday'); 
+   break;
+  case 'Tuesday':
+    console.log('It\'s Tuesday');
+    break; // Use break key word if you don't use return key word in the statement;
+    .... cases for other days of the week
+  default:
+    console.log('Unknow day');
+    break;
+}
+*/
+
+const today = "Tuesday";
+
+if (today === "Tuesday") {
+  console.log("Today is Tuesday!");
+} else if (today === "Tuesday") {
+  console.log("Today is Tuesday ??");
+} else {
+  console.log("Today is not Tuesday and not Sunday!");
+}
+
+// 1. Write a function that takes in a number and checks if it's even or odd using if else statement.
+
+function isEvenOrOdd(number) {
+  if (number % 2 === 0) {
+    return "Even";
+  } else {
+    return "Odd";
+  }
+}
+console.log(isEvenOrOdd(-27));
+console.log(isEvenOrOdd(52));
+
+// 2. Write a function that takes age as input. If the age is less than 18, print
+// "Sorry, you are not old enough to vote". If the age is between 18 and 65 (inclusive),
+// print "You are eligible to vote". If the age is greater than 65, print "You are a senior citizen".
+function checkVotingEligebility(age) {
+  if (age < 18) {
+    return "Sorry, you are not old enough to vote";
+  } else if (age >= 18 && age <= 65) {
+    // age >=18 AND age <=65 (Alternative to logical and is logical OR ||)
+    return "You are eligible to vote";
+  } else {
+    return "You are a senior citizen";
+  }
+}
+console.log(checkVotingEligebility(65));
+console.log(checkVotingEligebility(18));
+console.log(checkVotingEligebility(17));
+console.log(checkVotingEligebility(67));
+// 3. Write a function that asks the user to enter two numbers and checks if the first number is greater than, less than, or equal to the second number using if else statement.
+
+function compareNumbers(number1, number2) {
+  // You can skipp {} in if else statements if you only have *one line code* to execute in each case
+  if (number1 === number2) return "Numbers are equal.";
+  else if (number1 < number2) return "Number1 is less than Number2.";
+  else return "Number1 is greater than Number2.";
+}
+
+console.log(compareNumbers(1, 50));
+
+// 4. Write a function that takes in a string and checks if the first letter is uppercase using if else statement.
+function isFirstLetterUpperCase(string) {
+  const firstLetter = string.charAt(0); // string[0]
+  if (string.length <= 0 || typeof string === "String")
+    return "Invalid input value";
+
+  if (firstLetter === firstLetter.toUpperCase()) {
+    return true;
+  } else {
+    return false;
+  }
+}
+console.log(isFirstLetterUpperCase(""));
+console.log(isFirstLetterUpperCase("Write a function that"));
+
+// 5. Write a function that takes in a number and checks if it's positive, negative, or zero using if else statement.
+
+function checkPositiveOrNegative(conditionalNumber) {
```

Comment:
I think 'number' would be a better name here for the parameter (instead of conditionalNumber) since the value that the function expects is just a number.

Reply:
```suggestion
function checkPositiveOrNegative(number) {
```